specs/core/0_beacon-chain.md

```diff
@@ -1150,6 +1169,12 @@ def get_initial_beacon_state(initial_validator_deposits: List[Deposit],
         persistent_committees=[],
         persistent_committee_reassignments=[],
 
+        # Proof of custody
+        poc_challenges=[],
+        poc_commitment=ZERO_HASH,
```

Comment:
These 3 fields belong in the `ValidatorRecord` init.

Validators should actually make and remember their commitment in the deposit phase so that all of the validators don't have to submit a commitment at once when we switch to phase 1